README.md

```diff
@@ -3,18 +3,24 @@
 ![](https://visitor-badge.laobi.icu/badge?page_id=arbiter)
 ![Github Actions](https://github.com/primitivefinance/arbiter/workflows/Rust/badge.svg)
 
-> A stand-alone Rust analytic program for UniswapV3 pools and...
+> A stand-alone analytic program for UniswapV3 pools wirtten in rust.
 
 ## Motivation:
 
-The current implementation abstracts away worrying about fetching and generating token bindings for any users. Arbitrage takes advantage of price discrepancies in the buy and sell prices of different markets for the same assets. Right now, searchers can detect differences in reported prices between markets and execute pure profit[^1] trades, which we call _atomic arbitrage_. The environment for arbitrage is competitive, but for less popular token pairs, there is room for new searchers to equilibrate the prices of different markets.
+Arbiter is built to streamline insights, analysis and arbitrage simualtion for UniswapV3. The goal is to provide a tool that can be used to monitor activity on pools to measure PnL for liquidity providers and measure economic security of defi protocols. The program is designed to be used by anyone for whatever needs they desire. The current implementation abstracts away worrying about fetching and generating token bindings for any users.
 
-We want to develop a Rust program capable of detecting and executing arbitrage opportunities between exchanges. Our goal is to lower the barrier to entry for searching in hopes that it can help level the playing field for capturing arbitrage.
+Searchers detect differences in reported prices between markets and execute pure profit[^1] trades, which we call _atomic arbitrage_. The environment for arbitrage is competitive and it is time to streamline the simluation, and analysis of the implications of arbitrage.
 
 [^1]: 'Pure profit' refers to the fact that atomic arbitrage does not require arbitrageurs to put any capital at risk other than an upfront cost on building the strategy and gas cost to get their transaction included. This contributes to the lucrative nature of atomic arbitrage when done effectively
 
+## Architecture:
+
+There are two primary crates architect, which handles the construction and execution of transactions, and clairvoyance, which handles the monitoring of UniswapV3 pools. The program is designed to be modular and extensible.
```

Comment:
we should add single-line code blocks here to emphasize the fact they are crates - `architect` and `clairvoyance`

Reply:
nice i'm down